public/views/answers.html

```diff
@@ -1,89 +1,65 @@
+            <p class="valign" ng-bind-html-unsafe="card.text"></p>
+            <span ng-show="showFirst(card)"> 1 </span>
+            <span ng-show="showSecond(card)"> 2 </span>
+          </div>
         </div>
-    </span>
-
-  </div>
-</div>
-
-<div id="info-container" ng-show="game.state === 'awaiting players'">
-  <div id="inner-info">
-    <div id="lobby-how-to-play">How To Play</div>
-    <ol id="oh-el">
-      <li>Each player begins with, and will always have, 10 white answer cards.</li>
-      <li>For each round, one player is randomly chosen as the Card Czar.</li>
-      <li>Everyone else answers the black question card by clicking on the answer card they want to use.</li>
-      <li>The Card Czar then picks a favorite answer, and whoever played that answer wins the round.</li>
-      <li>*Want to adorn your avatar with a glorious crown? Donate to charity after the game!</li>
-    </ol>
+      </div>
+      <div ng-repeat="answer in game.table">
+        <div class="col s6 m4 l3" ng-style="{'cursor':'pointer'}" ng-click="pickWinning(answer)" ng-show="showTable || isCzar()" ng-repeat="card in answer.card" ng-animate="{enter:'animated bounceInLeft'}" style="background-color:{{winningColor($parent.$index)}}">
+          <div class="card hoverable">
+            <div class="card-content valign-wrapper an-answer">
+              <p class="valign" ng-bind-html-unsafe="card.text"></p>
+              <span ng-show="firstAnswer($index)"> 1 </span>
+              <span ng-show="secondAnswer($index)"> 2 </span>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
   </div>
-</div>
-<div id="game-end-container" ng-show="game.state === 'game ended' || game.state ==='game dissolved'">
-  <div id="inner-info-exit">
-    <div class="game-end-answer-text" ng-show="game.state ==='game dissolved' && game.gameWinner === -1">
-      <h2>What would you like to do now?</h2>
-      You might consider donating to charity to atone for the horrifying answers you've submitted. Donating will also place a lovely crown on your avatar's head.
+  <div ng-show="game.state === 'awaiting players'">
+    <h5>INSTRUCTIONS</h5>
+    <div>
+      <ul>
+        <li>Each player begins with, and will always have, 10 white answer cards.</li>
+        <li>For each round, one player is randomly chosen as the Card Czar.</li>
+        <li>Everyone else answers the black question card by clicking on the answer card they want to use.</li>
+        <li>The Card Czar then picks a favorite answer, and whoever played that answer wins the round.</li>
+      </ul>
     </div>
-    <div class="game-end-answer-text" ng-show="game.state === 'game ended' || game.gameWinner !== -1">
-      <h2>What would you like to do now?</h2>
+  </div>
+  <div ng-show="makeDecision('noGame')">
+    <h4 class="center-align">What would you like to do now?</h4>
+    <p class="center-align" ng-show="game.state ==='game dissolved' && game.gameWinner === -1">
```

Comment:
Looks like you missed this. Might want to replace it with `makeDecision('gameEndPeopleLeft')`